strings-csharp/SubstringInString/SubstringInString/Program.cs

```diff
@@ -1,7 +1,9 @@
 ﻿using BenchmarkDotNet.Running;
+using System.Text.RegularExpressions;
 
 var str = "Lorem ipsum dolor sit amet, consectetur adipiscing elip. Duis quis nisip eget sem vehipula accumsan";
 var toFind = "ip";
+var regex = new Regex(toFind, RegexOptions.Compiled); 
```

Comment:
We don't necessarily need to make it a compiled Regex here, just for the benchmark to have the highest performance we can give it. This also applies in the tests. There is a startup cost to compiling the Regex.